### PR TITLE
Mira

### PR DESCRIPTION
**Pull request by LikaLex**

### CODE REVIEW

File: app/controllers/clients_controller.rb [hunk 1]
```diff
@@ -44,6 +47,13 @@ def find_client
   end
 
   def client_params
-    params.require(:client).permit(:first_name, :second_name,  :phone_number)
+   # params.require(:client).permit(:first_name, :second_name,  :phone_number, :birth,
```
Comment (Aleksei12942) [inline app/controllers/clients_controller.rb]: Please delete all useless comments

File: app/controllers/metrics_controller.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+class MetricsController < ApplicationController
+  before_action :find_metric, only: [:show]
+  include ClientableControllerConcern
+
+  def index
+    @metrics = @client.metrics.order('created_at DESC')
+  end
```
Comment (Equinox-01) [inline app/controllers/metrics_controller.rb]: Add empty space

File: app/controllers/snapshots_controller.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
+class SnapshotsController < ApplicationController
+  include ClientableControllerConcern
+
+  before_action :find_snapshot, only: [:show]
+
+  def index
+    @snapshots = @client.snapshots.paginate(page: params[:page], per_page: 10)
+  end
+
+  def show
```
Comment (Equinox-01) [inline app/controllers/snapshots_controller.rb]: Use compact style

File: app/controllers/metrics_controller.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  def show
```
Comment (Equinox-01) [inline app/controllers/metrics_controller.rb]: Use compact style
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/EmptyMethod

File: config/routes.rb [hunk 5]
```diff
@@ -4,9 +4,11 @@
       root to: "users#index"
     end
   root to: 'visitors#index'
-  get 'products/:id', to: 'products#show', :as => :products
   devise_for :users
   resources :users
   resources :trainings
```
Comment (Equinox-01) [inline config/routes.rb]: Remove all unused routes

File: config/routes.rb [hunk 6]
```diff
@@ -4,9 +4,11 @@
...
-  resources :clients
+  resources :clients do
+    resources :metrics, shallow: true
+    resources :snapshots, shallow: true
+  end
 end
```
Comment (Equinox-01) [inline config/routes.rb]: Remove unused routes

File: app/views/clients/stats.html.slim [hunk 7]
```diff
@@ -0,0 +1,56 @@
+== javascript_include_tag 'tester'
```
Comment (Aleksei12942) [inline app/views/clients/stats.html.slim]: Do you really need it here?

File: app/controllers/metrics_controller.rb [hunk 8]
```diff
@@ -0,0 +1,40 @@
+class MetricsController < ApplicationController
+  before_action :find_metric, only: [:show]
+  include ClientableControllerConcern
+
+  def index
+    @metrics = if @client
```
Comment (brutu33) [inline app/controllers/metrics_controller.rb]: @metrics = @client ? @client.metrics : Metric.all

**Pull request merged**